CHANGELOG.rst

```diff
@@ -2,6 +2,16 @@
 Changelog for package urdfdom_py
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+1.0.0 (2020-01-06)
```

Comment:
Don't make any changes to the CHANGELOG by hand; that will get done automatically when I do a release.